### DISCUSSION TITLE
Informacja o zmianie prowizji w danej kategorii

### DISCUSSION THREAD

**Discussion by pgasidlo** [allegro/allegro-api#5403] [Suggestions - rejected]
Przydałoby się dodanie do `GET /sale/category-events` nowego zdarzenia: `CATEGORY_COMMISSION` informującego, że zmieniła się stawka prowizji dla tej kategorii.

**Comment by MartaNowaczyk** [comment]
Po weryfikacji zespołu odpowiedzialnego za ten zasób, sugestię muszę odrzucić. Powodem jest zbyt duża złożoność rozwiązania by osiągnąć opisany rezultat, przy bardzo niskim zainteresowaniu.
